src/api/tests/service/tts/test_minimax_http_streaming.py

```
@@ -0,0 +1,263 @@
+            (audio_data, int(kwargs.get("end_ms") or 1000))
+            if kwargs.get("end_ms") is not None
+            else (b"", 0)
+        ),
+    )
+    monkeypatch.setattr(
+        "flaskr.service.tts.streaming_tts.concat_audio_best_effort",
+        lambda parts, output_format="mp3": b"".join(parts),
+    )
+    monkeypatch.setattr(
+        "flaskr.service.tts.streaming_tts.get_audio_duration_ms",
+        lambda _audio, format="mp3": 1000,
```

Ruff (0.15.10)

[error] Lambda argument `format` is shadowing a Python builtin

(A006)